src/js/multiDomainFirstParties.js

```diff
@@ -181,7 +168,7 @@ var _multiDomainFirstPartiesArray = [
   ["walmart.com", "wal.co"],
   ["wellsfargo.com", "wf.com"],
   ["wikia.com", "wikia.net", "nocookie.net"],
-  ["wikipedia.org", "wikimedia.org", "wikimediafoundation.org", "wiktionary.org",
+  ["wfmlabs.org","wikipedia.org", "wikimedia.org", "wikimediafoundation.org", "wiktionary.org",
```

Comment:
Typo: should be `wmflabs.org`!

Nit: missing a separating space. Plus, could we move it to the back of the list? We should have the most important sites up front probably.